<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="UTF-8">
    <title>Ning Leng&#39;s GitHub page by lengning</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
  </head>
  <body>
    <section class="page-header">
      <h1 class="project-name">Ning Leng&#39;s GitHub page</h1>
      <h2 class="project-tagline"></h2>
    </section>

    <section class="main-content">
      <h2>
<a id="software" class="anchor" href="#software" aria-hidden="true"><span class="octicon octicon-link"></span></a>Software</h2>

<h3>
<a id="ebseq" class="anchor" href="#ebseq" aria-hidden="true"><span class="octicon octicon-link"></span></a>EBSeq</h3>

<p>EBSeq is an R package for identifying differentially expressed genes and isoforms using RNA-seq data. EBSeq provides improved power and well-controlled FDR in isoform analysis by accounting for varying uncertainty across different isoform groups. It accommodates
experiments with two or more conditions. EBSeq graphical user interface and Galaxy wrapper are also available on EBSeq page.</p>

<p><a href="https://www.biostat.wisc.edu/%7Ekendzior/EBSEQ/">EBSeq page</a></p>

<p><a href="https://github.com/lengning/EBSeq">EBSeq on GitHub</a></p>

<h3>
<a id="ebseqhmm" class="anchor" href="#ebseqhmm" aria-hidden="true"><span class="octicon octicon-link"></span></a>EBSeqHMM</h3>

<p>EBSeqHMM is an R package for detecting expression changes in RNA-seq experiments with ordered conditions (e.g. time or spatial course). EBSeqHMM incorporates a hidden markov component to accommodate dependence between adjacent time points (positions).</p>

<p><a href="https://www.biostat.wisc.edu/%7Ekendzior/EBSEQHMM/">EBSeqHMM page</a></p>

<p><a href="https://github.com/lengning/EBSeqHMM">EBSeqHMM on GitHub</a></p>

<h3>
<a id="oefinder" class="anchor" href="#oefinder" aria-hidden="true"><span class="octicon octicon-link"></span></a>OEFinder</h3>

<p>Identify genes with artificial expression trends in single cell RNA-seq data (ordering effect genes) - with GUI</p>

<p><a href="https://github.com/lengning/OEFinder">OEFinder on GitHub</a></p>

<h3>
<a id="oscope" class="anchor" href="#oscope" aria-hidden="true"><span class="octicon octicon-link"></span></a>Oscope</h3>

<p>Identify oscillatory genes in unsynchronized single cell RNA-seq data</p>

<p><a href="https://www.biostat.wisc.edu/%7Ekendzior/OSCOPE/">Oscope page</a></p>

<h3>
<a id="gene-set-enrichment-analysis-gui" class="anchor" href="#gene-set-enrichment-analysis-gui" aria-hidden="true"><span class="octicon octicon-link"></span></a>Gene set enrichment analysis GUI</h3>

<p><a href="https://github.com/lengning/Enrichment_shiny">on GitHub</a></p>

<h3>
<a id="scpattern" class="anchor" href="#scpattern" aria-hidden="true"><span class="octicon octicon-link"></span></a>SCPattern</h3>

<p>Identify expression changes in single cell experiment with two or more conditions</p>

<p><a href="https://github.com/lengning/SCPattern">SCPattern on GitHub</a></p>

<h2>
<a id="publications" class="anchor" href="#publications" aria-hidden="true"><span class="octicon octicon-link"></span></a>Publications</h2>

<h3>
<a id="selected-publications" class="anchor" href="#selected-publications" aria-hidden="true"><span class="octicon octicon-link"></span></a>Selected Publications</h3>

<ul>
<li>  N. Leng<em>, L. Chu</em>, C. Barry, Y. Li, J. Choi, X. Li, P. Jiang, R. Stewart, J.A. Thomson, and C. Kendziorski, Oscope identifies oscillatory genes in unsynchronized single cell RNA-seq. Nature Methods, 2015 (Press in Science Daily, Health Canal, Morningstar, Sleep Review, Phys.org, etc.)</li>
<li>  N. Leng, Y. Li, B.E. Mcintosh, B. K. Nguyen, B. Duffin, S. Tian, J. A. Thomson, C. Dewey, R. Stewart, and C. Kendziorski. EBSeq-HMM: A Bayesian approach for identifying gene-expression changes in ordered RNA-seq experiments. Bioinformatics, 2015</li>
<li>  N. Leng, J.A. Dawson, J.A. Thomson, V. Ruotti, A.I. Rissman, B.M.G. Smits, J.D. Haag, M.N. Gould, R.M. Stewart, and C. Kendziorski. EBSeq: An empirical Bayes hierarchical model for inference in RNA-seq experiments. Bioinformatics, 2013 (Featured highly-cited article from Bioinformatics in 2014)</li>
<li>  N. Leng, J. Choi, L. Chu, J. Thomson, C. Kendziorski, R. Stewart, OEFinder: A user interface to identify and visualize ordering effects in single cell RNA-seq data. Under review.</li>
</ul>

<h3>
<a id="other-publications" class="anchor" href="#other-publications" aria-hidden="true"><span class="octicon octicon-link"></span></a>Other Publications</h3>

<ul>
<li>  K.A. Tran, S.A. Jackson, Z.PG. Olufs, N. Leng, C. Kendziorski, S.Roy and R. Sridharan, Temporal acquisition of epigenetic and transcriptional modulation during cell fate change. Nature Communications, 2015.</li>
<li>  T. Zhang, A. Patro, N.Leng and S. Banerjee, A Wireless Spectrum Analyzer in Your Pocket. ACM Hotmobile, 2015</li>
<li>  T. Zhang, N. Leng, S. Banerjee, A Vehicle-based Measurement Framework for Enhancing Whitespace Spectrum Databases, ACM Mobicom 2014 (Best presentation award of Mobicom 2014)</li>
<li>  E.A. Hines, R.J. Szakaly, N. Leng, J.M. Verheyden; A.T. Webster, C. Kendziorski, L.A. Rosenthal, J.E. Gern, R.L. Sorkness, X. Sun, R.F. Lemanske, Comparison of Temporal Transcriptomic Profiles from Immature Lungs of Two Rat Strains Reveals a Viral Response Signature Associated with Chronic Lung Dysfunction, Plos ONE, 2014</li>
<li>  H.C. St. John, K.A. Bishop, M.B. Meyer, N.A. Benkusky, N. Leng, C. Kendziorski, L. Bonewald and J.W. Pike, The Osteoblast to Osteocyte Transition: Epigenetic Changes and Response to the Vitamin D3 Hormone. Molecular Endocrinology, 2014</li>
<li>  C. Yao, N. Leng, K. Weigel, K.E. Lee, C.D. Engelman and K.J. Meyers. Prediction of Genetic Contributions to Complex Traits using Whole Genome Sequencing Data. BMC Proceedings, 2014</li>
<li>  R. Stewart, C.A. Rasc√≥n, S. Tian, J. Nie, M.D. Probasco, J.M. Bolin, H. Ardalani, R.J. Wagner, L.F. Chu, C. Barry, N. Leng, S. Sengupta, M. Volkmer, B. Habermann, E.M. Tanaka, J.A. Thomson, C.N. Dewey. Comparative RNA-seq Analysis in the Unsequenced Axolotl: The Oncogene Burst Highlights Early Gene Expression in the Blastema. Plos Computational Biology, 2013</li>
<li>  B.M. G. Smits, J.D. Haag, A.I. Rissman, D. Sharma, A. Tran, A.A. Schoenborn, R.C. Baird, D.S. Peifer, D.Q. Leinweber, M.J. Muelbl, M.L. Amanda, M.R. Eichelberg, N. Leng, C. Kendziorski, M.C. John, P.A. Powers, C.M. Alexander, M.N. Gould. The Gene Desert Mammary Carcinoma Susceptibility Locus Mcs1a Regulates Nr2f1 Modifying Mammary Epithelial Cell Differentiation and Proliferation. Plos Genetics, 2013</li>
</ul>

      <footer class="site-footer">

        <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
      </footer>

    </section>

  
  </body>
</html>
